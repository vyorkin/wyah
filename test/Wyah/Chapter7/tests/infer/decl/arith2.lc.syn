Decl "nsucc" (ELam (Var "x") (EOp Add (EVar (Var "x")) (ELit (LInt 1))))
