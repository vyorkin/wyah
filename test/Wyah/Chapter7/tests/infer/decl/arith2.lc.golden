Forall [] (TCon "Int" :-> TCon "Int")